packages/sync/src/class-functions.php

```diff
@@ -161,15 +161,21 @@ public static function get_post_type_features() {
 	 * @return string Hosting provider.
 	 */
 	public static function get_hosting_provider() {
-		if ( defined( 'GD_SYSTEM_PLUGIN_DIR' ) || class_exists( '\\WPaaS\\Plugin' ) ) {
-			return 'gd-managed-wp';
-		}
-		if ( defined( 'MM_BASE_DIR' ) ) {
-			return 'bh';
-		}
-		if ( defined( 'IS_PRESSABLE' ) ) {
-			return 'pressable';
+		$hosting_provider_constants = array(
+			'GD_SYSTEM_PLUGIN_DIR' => 'gd-managed-wp',
+			'MM_BASE_DIR'          => 'bh',
+			'PAGELYBIN'            => 'pagely',
+			'KINSTAMU_VERSION'     => 'kinsta',
+			'FLYWHEEL_CONFIG_DIR'  => 'flywheel',
+			'IS_PRESSABLE'         => 'pressable',
+		);
+
+		foreach ( $hosting_provider_constants as $constant => $constant_value ) {
```

Comment:
Instead of `$constant_value`, perhaps we can use `$hosting_provider`?

Reply:
Makes more sense. I've updated this